DigitalLearningSolutions.Web/ViewModels/Register/RegisterViewModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.Register
+{
+    using System.ComponentModel.DataAnnotations;
+
+    public class RegisterViewModel
+    {
+        [Required(ErrorMessage = "Please enter a first name.")]

```

Comment:
I'm wondering if we should use error messages in the NHS example format (I remember reading that Please shouldn't be used, maybe it was in govuk website):
![image](https://user-images.githubusercontent.com/40016796/115696635-ca383f00-a35a-11eb-9cec-1e522e3cd354.png)

Reply:
Here it is: https://design-system.service.gov.uk/components/error-message/#be-clear-and-concise

(I don't think we follow it everywhere, but might be good to try and do it as much as possible)